BinaryEncoding.md

```diff
@@ -335,9 +332,9 @@ The `br_table` operator has an immediate operand which is encoded as follows:
 
 | Field | Type | Description |
 | ---- | ---- | ---- |
-| target_count | `uint16` | number of targets in the target_table |
-| target_table | `uint16*` | target entries that indicate an outer block or loop to which to break |
-| default_target | `uint16` | an outer block or loop to which to break in the default case |
+| target_count | `varuint32` | number of targets in the target_table |
+| target_table | `varuint32*` | target entries that indicate an outer block or loop to which to break |
```

Comment:
Oh, hah, sorry, I didn't mean to ignore it.  It just sounds like more work to validate and then we'd need some irregular (compared to everything else) spec language to explain it.  `uint32` seems like the least-bad compromise between these competing constraints, imho.  Happy to revisit later in a separate issue, though, just hoping not to hang up this one.